src/HtmlExtension.php

```diff
@@ -108,4 +109,22 @@ function twig_html_classes(...$args): string
 
     return implode(' ', array_unique($classes));
 }
+
+function twig_html_attributes(array $attributes): string
+{
+    $output = '';
+
+    foreach ($attributes as $attribute => $value) {
+        if ($value === true) {
+            $output .= ' ' . $attribute;
```

Comment:
the attribute name must also be escaped to be safe (and using the `html_attr` escaping as it is not quoted)

Reply:
how can I access the escape('html_attr') function inside extensions?
I would suggest to simply escape chars like <,>,=,"
what do you think?